README.md

```diff
@@ -21,7 +21,49 @@ theme: minima
 And then execute:
 
     $ bundle
+
     
+## Contents At-A-Glance
+
+Minima has been scaffolded by the `jekyll new-theme` command and therefore has all the necessary files and directories to have a new Jekyll site up and running with zero-configuration.  
```

Comment:
Minima **is** scaffolded sounds like its continuously scaffolded.. like e.g. "The President is supported by his ministers.."
**has been**  scaffolded makes it an event in the near past. e.g. "The Asset has been compromised.."